docs/aws/deploy-heroku.md

```diff
@@ -0,0 +1,34 @@
+## How to setup a heroku app with cloudfront
+
+Assuming you have created your heroku app and have the link ready e.g. (appname.herokuapp.com)
+
+### Create Cloudfront Distribution
+
+1. Open cloudfront and hit **Create Distribution**
+2. In this case we are going to click **Get Started** on create web distribution.
+3. Fill in the blank for **Origin Settings section**.  
+    3.1 Origin Domain Name: This should be your heroku URL (appname.herokuapp.com)
+    3.2 Origin Path: This should be your base route for example: **appname.herokuapp.com/home** then this should be **/home**
+    3.3 Origin ID: This will be auto generated.
+    3.4 Origin Custom Headers: You don't need this unless you need to add header in every request.
+    3.5 
+4. Fill in the blank for **Default Cache Behavior Settings**.   
+    4.1 Viewer Protocol Policy: Redirect HTTP to HTTPS if you need this to be on HTTPS  
+    4.2 Allowed HTTP Methods: Leave it **GET, HEAD** unless you need other METHODS.  
+    _NOTE: leave other part as default in this section unless you need something custom_  
+5. Fill in the blank for **Distribution Settings**  
+    5.1 Alternate Domain Names (CNAMEs): This should be your Route53 domain name for example: appname.mofostaging.net  
+    5.2 SSL Certificate: You would probably want **Custom SSL Certificate (example.com):** and choose the one apply to your domain, in this case **mofostaging.net** is the one here.  
```

Comment:
"You would probably want **Custom SSL Certificate (example.com):**. Select the one that matches your Route53 domain. "